src/Core/src/Core/IPage.cs

```diff
@@ -3,7 +3,7 @@ namespace Microsoft.Maui
 	/// <summary>
 	/// A View that occupies the entire screen.
 	/// </summary>
-	public interface IPage : IView
+	public interface IPage : IFrameworkElement
```

Comment:
Important changes: Page should be a FrameworkElement instead a View. We will not have gestures and other functionality at App or Page level.